src/OpenSearch.Net/Responses/ServerException/ErrorCause.cs

```diff
@@ -190,9 +187,6 @@ public TErrorCause Deserialize(ref JsonReader reader, IJsonFormatterResolver for
 								case 8:
 									errorCause.Language = reader.ReadString();
 									break;
-								case 9:
-									errorCause.LicensedExpiredFeature = reader.ReadString();
-									break;
```

Comment:
Similar question as we have now a break in the cases. I just don't know if it's OK but wanted to ensure this was checked.

Reply:
Updated the changes to adjust the numbers downstream.